src/Pages/form/Components/ArrivalOnset.tsx

```diff
@@ -63,55 +68,70 @@ const OnsetStyledToggleButtonGroup = withStyles(() => ({
   },
 }))(ToggleButtonGroup);
 
-const ArrivalOnset: React.FC = () => {
+interface PatientProps {
+  arrivalDate: Date | null;
+  arrivalTime: Date | null;
+  clearDate: Date | null;
+  clearTime: Date | null;
+  lastDate: Date | null;
+  lastTime: Date | null;
+  firstDate: Date | null;
+  firstTime: Date | null;
+  onset: string;
+}
+
+interface Props {
+  values: PatientProps;
+  fieldName: string;
+  onChange: (field: string, value: any, shouldValidate?: boolean) => void;
```

Comment:
Should not be any

Reply:
I try to use `Date` because all values in these components are dates but I got this error. 
![image](https://user-images.githubusercontent.com/59549784/130347720-1e959151-7ed0-444d-a473-dcdcf9a9e846.png)

Ps. Steel and Fern also use `any` they told me that this line is from Formik docs.